explorer/src/app/globals.css

```
@@ -0,0 +1,95 @@
+tbody tr {
+  transition: background-color 0.15s ease;
+}
+
+tbody tr:hover {
+  background-color: #f8fafc;
+}
+
+/* Focus styles */
+input:focus, select:focus, button:focus {
+  outline: none;
+  ring: 2px;
```

Biome (2.1.2)

[error] Unknown property is not allowed.

See CSS Specifications and browser specific properties for more details.
To resolve this issue, replace the unknown property with a valid CSS property.

(lint/correctness/noUnknownProperty)